.github/workflows/rewrite-kernel-configs.yml

```
@@ -0,0 +1,292 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Download all change artifacts (no merge)
+        uses: actions/download-artifact@v4
+        with:
+          path: _artifacts
+          pattern: changes-*
+          if-no-files-found: ignore
```

actionlint (1.7.7)

input "if-no-files-found" is not defined in action "actions/download-artifact@v4". available inputs are "github-token", "merge-multiple", "name", "path", "pattern", "repository", "run-id"

(action)